__test__/TreeDonation.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import TreeDonation from '../src/features/donations/screens/TreeDonation';
+import { project } from './mockData/projectData';
+
+
+describe('tree donation', () => {
```

Comment:
What's the purpose of this file - it does not run as a test yet and imports some mock data not used otherwise?

Reply:
If it should be a template for demonstrating how to write future tests I would be fine to keep it. Maybe add a comment to describe the purpose!